pytest_plugins/issue_handlers.py

```diff
@@ -42,14 +66,39 @@ def pytest_collection_modifyitems(session, items, config):
 
     pytest.issue_data = generate_issue_collection(items, config)
 
-    # No item deselect, just adding skipif markers
+    # Add skipif markers, and modify collection if there were BZ options passed
+    bz_filters = config.getoption('BZ', None)
+    selected = []
+    deselected = []
     for item in items:
+        # Add a skipif marker for the issues
         skip_if_open = item.get_closest_marker('skip_if_open')
         if skip_if_open:
             # marker must have `BZ:123456` as argument.
             issue = skip_if_open.kwargs.get('reason') or skip_if_open.args[0]
             item.add_marker(pytest.mark.skipif(is_open(issue), reason=issue))
 
+        # remove items from collection
+        if bz_filters:
+            # Only include items which have BZ mark that includes any of the filtered bz numbers
+            item_marked_bzs = getattr(item.get_closest_marker('BZ', None), 'args', [])
```

Comment:
Can we rename this var to something better ?

Something  like bz_marked_items ?

Reply:
`bz_marked_items` is misleading, as its not a list of items with BZ markers. Its a list of the BZs marked on the given item.  Changing to `item_bz_marks`.